tsdb/engine/tsm1/cache.go

```diff
@@ -344,6 +344,7 @@ func (c *Cache) DeleteRange(keys []string, min, max int64) {
 		c.store[k].filter(min, max)
 		if c.store[k].count() == 0 {
 			delete(c.store, k)
+			continue
```

Comment:
This will skip the `c.size` decrement below.  Should add a `c.size -= uint64(origSize)` before `continue`